Cargo.toml

```diff
@@ -12,52 +12,42 @@ edition = "2018"
 publish = false
 
 [features]
-default = ["jemalloc"]
```

Comment:
Why remove it?

Reply:
The default is set in cmd/Cargo.toml, so we still have the same defaults. If we try and set it here, then all the crates which depend on TiKV need to set it because they must now use `default-features = false`

Reply:
Does this mean that using `cargo bench ...` will no longer contain jemalloc. Instead, you have to use `cargo bench --features jemalloc`?